### DISCUSSION TITLE
Is this the expected behavior of the inset-0 class?

### DISCUSSION THREAD

**Discussion by armandoborge** [tailwindlabs/tailwindcss#10824] [Help]
I am expecting the black box (child) to be centered inside the gray box (parent).
Am I missing something here?

https://play.tailwindcss.com/ExFSJh3ETv

**Comment by wongjn** [comment]
`inset-0` applies
```css
top: 0px;
right: 0px;
bottom: 0px;
left: 0px;
```
which one should not expect to center the child inside the parent.

**Comment by armandoborge** [comment]
I was misunderstanding the example in the docs: https://tailwindcss.com/docs/top-right-bottom-left#placing-a-positioned-element
So nevermind, thanks for replying
